src/mainwindow.ui

```diff
@@ -195,6 +195,80 @@
       <property name="currentIndex">
        <number>0</number>
       </property>
+      <widget class="QWidget" name="sendMsg_Tab">
+       <attribute name="title">
+        <string>Send Message</string>
+       </attribute>
+       <layout class="QFormLayout" name="formLayout">
+        <property name="formAlignment">
+         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
+        </property>
+        <item row="0" column="0" colspan="2">
+         <layout class="QHBoxLayout" name="horizontalLayout_5">
+          <item>
+           <widget class="QLineEdit" name="sendMsg_TextBox">
+            <property name="placeholderText">
+             <string>Enter message to send</string>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QPushButton" name="sendMsg_Button">
+            <property name="sizePolicy">
+             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
+              <horstretch>0</horstretch>
+              <verstretch>0</verstretch>
+             </sizepolicy>
+            </property>
+            <property name="text">
+             <string>Send</string>
+            </property>
+           </widget>
+          </item>
+         </layout>
+        </item>
+        <item row="1" column="0" colspan="2">
+         <layout class="QHBoxLayout" name="horizontalLayout_6">
+          <item>
+           <widget class="QCheckBox" name="clearOnSendCheckBox">
+            <property name="text">
+             <string>Clear after send</string>
+            </property>
+            <property name="checked">
+             <bool>true</bool>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QCheckBox" name="sendOnReturnCheckBox">
+            <property name="text">
+             <string>Send on return/enter press</string>
```

Comment:
Just simplify this to "Send on Enter."  It's called "Return" only on macOS, but I don't know of an easy way to change this text depending on the platform.  I know of several difficult ways, but I doubt they're worth the effort.